### PR TITLE
Clear all

### PR DESCRIPTION
**Pull request by StarMindz**
I added  a new JavaScript file and import it as a module:
The module contains methods related to the status updates (completed: true/false).
I also added an event listener to the checkbox (change) to implement a function for the "Clear all completed" button (use filter() method).
And store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/modules/storage.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+// document.getElementById('clearlist-btn').addEventListener('click', () => {
+//   tasksObject.taskList.filter((value) => value.completed !== true);
+//   tasksObject.taskList = tasksObject.taskList.map((todo, id) => ({
+//     index: id,
+//     description: todo.description,
+//     completed: todo.completed,
+//   }));
+//   dataStore();
+//   document.location.reload();
+// });
```
Comment (FrederickMih) [inline src/modules/storage.js]: - [ ] Comments make our codebase looks messy. Do not commit commented codes.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <input
+          id="to-do-list-input"
+          type="text"
+          name="List-name"
+          placeholder="Add to your list..."
+        />
+        <i id="add-button" class="input-icon fas fa-hand-point-left"></i>
+      </div>
+      <ul id="to-do-list"></ul>
+      <button type="submit" id="clearlist-btn">Clear all completed</button>
```
Comment (FrederickMih) [inline src/index.html:33]: - [ ] Based on the requirements of this project, a function for "Clear all completed" button using Array.filter() method should be implemented. Kindly fix this.

File: src/modules/storage.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+const tasksObject = new Tasks();
+const tasks = document.getElementById('to-do-list');
+export default tasksObject;
+
+const dataStore = () => {
+  localStorage.setItem('data', JSON.stringify(tasksObject.taskList));
+};
+
+export const getCheckers = () => {
```
Comment (FrederickMih) [inline src/modules/storage.js:11]: - [ ] When I complete a task by clicking on a checkbox and reloading the page, the task becomes uncompleted(uncheck.). Let's kindly ensure that all actions are updated in the local storage.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @StarMindz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- PR has a title and summary :heavy_check_mark: 
- No Linter error :heavy_check_mark: 
- There's an event listener on the checkbox.

## Required Changes ♻️


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by StarMindz** [comment]
@FrederickMih, Thank you for your review. All needed changes have been made

**Comment by StarMindz** [comment]
@FrederickMih, Thank you for your review. All needed changes have been made

**Review by mugishasam123 — APPROVED** [review]
Hi @StarMindz  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors 👍 
- Good commit messages 💯 
- Nice use of git-flow 🥇 

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**